src/macaron/slsa_analyzer/checks/build_as_code_check.py

```diff
@@ -114,6 +132,67 @@ def run_check(self, ctx: AnalyzeContext, check_result: CheckResult) -> CheckResu
                 # Checking if a CI service is discovered for this repo.
                 if isinstance(ci_service, NoneCIService):
                     continue
+
+                trusted_deploy_actions = defaults.get_list("builder.pip.ci.deploy", "github_actions", fallback=[])
+
+                # Check for use of a trusted Github Action to publish/deploy.
```

Comment:
```suggestion
                # Check for use of a trusted Github Actions workflow to publish/deploy.
```

Reply:
Fixed in [9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d).